src/librustdoc/clean/inline.rs

```diff
@@ -122,10 +122,16 @@ crate fn try_inline(
 
     let target_attrs = load_attrs(cx, did);
     let attrs = box merge_attrs(cx, Some(parent_module), target_attrs, attrs_clone);
+    let cfg = if let Some(new_attrs) = attrs_clone {
+        let mut both = new_attrs.to_vec();
+        both.extend_from_slice(target_attrs);
+        both.cfg(cx.sess().diagnostic())
+    } else {
+        target_attrs.cfg(cx.sess().diagnostic())
+    };
```

Comment:
`attrs` is of type `clean::Attributes`. The whole point of this series of changes was to take `cfg` out of `clean::Attributes`, so we cannot do `attrs.cfg()` anymore. It's now either stored in `Item` or computed on the run from `[ast::attributes]`. 
So here we want to compute them from attributes in order to store them in the `Item` we are building and we are facing the case where `clean::Attributes` is build from two sets of `[ast::attribute]` (`attrs` and `additional_attrs` as they are named in `clean::Attributes::from_ast`).

In `from_ast` the order was relevant, `additional_attrs` must be placed before `attrs` in the resulting `Attributes`, but I realize now that if we just compute `cfg` the order may not matter anymore. But I'm not entirely sure because it will influence the order in which the different `Cfg` are pushed into the `Cfg::All` vector. Does it matter latter ?

Reply:
> In from_ast the order was relevant, additional_attrs must be placed before attrs in the resulting Attributes, but I realize now that if we just compute cfg the order may not matter anymore. 

Correct, the order doesn't matter - the only things you can do with `Cfg` is logical AND and OR.

> The whole point of this series of changes was to take cfg out of clean::Attributes, so we cannot do attrs.cfg() anymore. It's now either stored in Item or computed on the run from [ast::attributes].

Ok I see - and adding this function to clean::Attributes makes it confusing (because now there are 3 different places you could look up the cfg, but you should use `Item::cfg` wherever possible. Maybe it makes sense to return the cfg of both attributes from `merge_attrs` directly? Then you get rid of the duplication and it's also more clear why this is separate from everything else.

Reply:
> Maybe it makes sense to return the cfg of both attributes from merge_attrs directly? Then you get rid of the duplication and it's also more clear why this is separate from everything else.

Yes it makes sense. I'm on it

Reply:
It's done 👍